modules/cnf-rfhe-notifications-api-refererence.adoc

```diff
@@ -0,0 +1,276 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+[id="cnf-rfhe-notifications-api-refererence_{context}"]
+= Redfish hardware event notifications REST API reference
+
+You can use the Redfish hardware fast event notifications REST API to subscribe an application to the Redfish hardware events for the parent node.
```

Comment:
remove "for the parent node" to avoid confusion. This is true since we are supporting SNO (single node openshift) for now, but technically DU application and Redfish hardware could be located in different nodes.